task/02-numbers-tasks.js

```diff
@@ -127,7 +128,8 @@ function getAngleBetweenVectors(x1, y1, x2, y2) {
  *     0     => 0
  */
 function getLastDigit(value) {
-    throw new Error('Not implemented');
+    let x = value.toString();
+	return Number(x.charAt(x.length - 1));
```

Comment:
Don't use `Number` to convert string to number)) Use unary `+` or `parseFloat/parseInt`.